assembl/static2/js/app/components/debate/navigation/timelineSegment.jsx

```diff
@@ -15,14 +12,9 @@ class TimelineSegment extends React.Component {
     const { isStepCompleted, isCurrentPhase, phaseIdentifier } = this.props;
     const { debateData } = this.props.debate;
     const slug = { slug: debateData.slug };
+
     if (isStepCompleted || isCurrentPhase) {
       browserHistory.push(`${get('debate', slug)}?phase=${phaseIdentifier}`);
-    } else {
-      const { locale } = this.props.i18n;
-      const startDate = getStartDatePhase(debateData.timeline, phaseIdentifier);
-      const phaseName = getPhaseName(debateData.timeline, phaseIdentifier, locale).toLowerCase();
-      const body = <div><Translate value="debate.notStarted" phaseName={phaseName} /><Localize value={startDate} dateFormat="date.format" /></div>;
-      displayModal(null, body, true, null, null, true);
     }
```

Comment:
The behaviour you expect is not part of the specification as of today, but I would be happy to discuss specification and make it more precise if you like :). And from my understanding, when I go to the `/{slug}/debate` page and current phase is a "thread" phase, I should indeed see the modal.